core/src/main/java/tc/oc/pgm/commands/StatCommands.java

```diff
@@ -0,0 +1,23 @@
+package tc.oc.pgm.commands;
+
+import app.ashcon.intake.Command;
+import net.md_5.bungee.api.ChatColor;
+import org.bukkit.command.CommandSender;
+import tc.oc.pgm.modules.StatsMatchModule;
+import tc.oc.util.bukkit.component.ComponentUtils;
+
+public class StatCommands {
+
+  @Command(
+      aliases = {"stats"},
+      desc = "Shows your stats for this match")
+  public static void checkStats(CommandSender sender) {
+    sender.sendMessage(
+        ComponentUtils.horizontalLineHeading(
+            ChatColor.DARK_GREEN + "Current stats",
```

Comment:
Should be placed inside the translations file.

Reply:
Of course! Great catch